fvtest/compilertriltest/ArithmeticTest.cpp

```diff
@@ -221,6 +221,31 @@ float fdiv(float l, float r) {
     return l/r;
 }
 
+float frem(float l, float r) {
+    float ans;
+    float anstrunc;
+    ans = l/r;
+    ans > 0.0 ? anstrunc = floorf(ans) : anstrunc = ceilf(ans);
+    if(anstrunc != ans)
+    {
+        if(ans + 0.5 > anstrunc + 1)
+        {
+            ans = anstrunc + 1;
+        }
+        else if(ans == anstrunc + 0.5)
+        {
+            (int)anstrunc % 2 == 0 ? ans = anstrunc : ans = anstrunc + 1;
+        }
+        else
+        {
+            ans = anstrunc; 
+        }
+    }
```

Comment:
I don't think this algorithm works. Testing it with `frem(-5.1, 3.0)` I get -2.1 instead of 0.9.

I'm still trying to investigate whether `remainderf`/`remainder` is supported on all our build platforms, so that option is not off the table yet. Still working on it though.

Reply:
Oops! Thanks for that, I think I found the issue. 
I may put this on hold until you're sure about remainderf(), since using that would be ideal, if we can. Could you send a message or make another comment when you do find out?

Reply:
Sorry @hsoontie. Still looking into it.

Reply:
After some investigation, I've concluded that `std::remainderf()` should work on all our supported compilers. Sorry for taking so long to reply.

Reply:
Awesome, thank you! I will revert it soon.

Reply:
@Leonardo2718 Done!